docs/src/administration/users.md

```diff
@@ -1,65 +1,53 @@
+sidebarTitle: "User administration"
 description: |
-  Every Platform.sh user has a role that controls their access and permission levels. Different roles allow different levels of access to your applications, environments and projects. You can manage how users interact with your project and environments at Platform.sh
+  Every Platform.sh user has a set of permissions. Depending on your role, you'll be able to access different levels of the application, environments and projects. If you want to add a user to a project or an environment type, the user has to create an account before they can contribute to the project.
 ---
 
 {{< description >}}
 
-Any user added to a project or an environment type on Platform.sh will need to [register for an account](https://auth.api.platform.sh/register) before they can contribute. If you need to delete your account at any time you can [transfer ownership](/administration/users.md#transfer-ownership) of your existing projects and then open a [support ticket](/development/troubleshoot.md#deleting-your-platformsh-account) to request your account deletion.  
+### User roles
 
-## User roles
+| User role    | Description |
+| ------------ |-------------|
+| Project owner                   | The organization where the project was created in the first place. There is only one and can create multiple project administrators. If you’re the owner of a project, and want to transfer it to a different user, first invite that user as a project administrator and then, submit a support ticket from your owner account to ask for the transfer. Transferring a project automatically transfers the upcoming subscription charges to the new owner. Once you transfer ownership, open a [support ticket](https://docs.platform.sh/development/troubleshoot.html) to request your account deletion. Note that transferring a project automatically transfers the upcoming subscription charges to the new owner. |
+|Project Admin | Users who can configure project settings, administer environment permissions, push code, and execute actions on all project environments.|                                                                                                                                                                                      
 
-Grant user permissions to the entire project:
-* **Project Viewer** - A project viewer can view all environments of the project.
-* **Project Administrator** - A project administrator can change settings, push code and execute actions on all environments of the project.
+### Permissions per environment type
 
-You can also define user access per environment type (Production, Staging, Development). Each permission level progressively increases access to the environments part of a given environment type:
+Environment types offer a way to group environments (production, staging, and development) together. If you group your environments, you can grant user permissions per environment type at the project level.
```

Comment:
Given the scope of the ticket (user confusion about types), I'd like to see the description of environment types somewhat expanded on. Just another sentence or two maybe. 

I also personally find the conjunction of "per environment type" with "at the project level" a little confusing. They seem to be pulling in different directions.